packages/hash/frontend/src/pages/[account-slug]/types/entity-type/entity-type-tabs.tsx

```diff
@@ -1,28 +1,82 @@
-import { EntityType } from "@blockprotocol/type-system-web";
 import { faPlus } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@hashintel/hash-design-system";
-import { Box, Typography } from "@mui/material";
-import { EntityTypeDefinitionTab } from "./entity-type-definition-tab";
+import { Box, Tabs, tabsClasses } from "@mui/material";
+import { useRouter } from "next/router";
+import { useMemo } from "react";
+import { useFormContext, useWatch } from "react-hook-form";
+import { EntityTypeEditorForm } from "./form-types";
 import { TabButton } from "./tab-button";
+import { TabLink } from "./tab-link";
+import { useEntityTypeEntities } from "./use-entity-type-entities";
+
+export const EntityTypeTabs = () => {
+  const router = useRouter();
+
+  const { control } = useFormContext<EntityTypeEditorForm>();
+  const propertiesCount = useWatch({ control, name: "properties.length" });
+
+  const { entities } = useEntityTypeEntities() ?? {};
+
+  const baseUri = useMemo(
+    () =>
+      `/${router.query["account-slug"]}/types/entity-type/${router.query["entity-type-id"]}`,
+    [router.query],
+  );
+
+  const currentTab = router.query.tab ?? "";
 
-export const EntityTypeTabs = ({ entityType }: { entityType: EntityType }) => {
   return (
-    <Box display="flex">
-      <EntityTypeDefinitionTab entityType={entityType} />
+    <Box display="flex" overflow="visible">
+      <Tabs
+        value={currentTab}
+        TabIndicatorProps={{
+          sx: ({ palette }) => ({
+            height: 3,
+            backgroundColor: palette.blue[60],
+            minHeight: 0,
+            bottom: -1,
+          }),
+        }}
+        sx={{
+          minHeight: 0,
+          overflow: "visible",
+          alignItems: "flex-end",
+          [`.${tabsClasses.scroller}`]: {
+            overflow: "visible !important",
+          },
+        }}
+      >
+        <TabLink
+          value=""
+          href={baseUri}
+          label="Definition"
+          count={propertiesCount}
+          active={currentTab === ""}
+        />
+        <TabLink
+          value="entities"
+          href={`${baseUri}?tab=entities`}
+          label="Entities"
+          count={entities?.length}
+          active={currentTab === "entities"}
+        />
+      </Tabs>
+
       <Box display="flex" ml="auto">
-        <TabButton href="#" sx={(theme) => ({ color: theme.palette.gray[90] })}>
-          <Typography variant="smallTextLabels" sx={{ fontWeight: 500 }}>
-            Create new entity
-          </Typography>
-          <FontAwesomeIcon
-            icon={faPlus}
-            sx={(theme) => ({
-              ...theme.typography.smallTextLabels,
-              color: theme.palette.blue[70],
-              ml: 1,
-            })}
-          />
-        </TabButton>
+        <TabButton
```

Comment:
Should Tablinks have an hover effect? If so I think the colors should be slightly different than those of the active tab, to avoid confusion. What do you think @nonparibus ?

Also fixed `TabButton`s height in 88137fc1143d8a85d32f7041906a98eac4233550.

Reply:
Sounds good. Can be 10 shade lighter; ty

Reply:
Added Tab hover effect in 2fd226bff74fa5fbb4b862ac9e9ff55d35a908fe.

![tab hover](https://user-images.githubusercontent.com/37453800/202774848-2c42d133-58df-45e6-9ace-132f75c97b7b.gif)